app/controllers/downloads_controller.rb

```diff
@@ -9,7 +9,12 @@ def latest
   end
 
   def guis
-    render "downloads/guis/index"
+    file = File.read('resources/guis.json')
+    parsed = JSON.parse(file)
+
+    guis_info = parsed["guis"]
+
+    render "downloads/guis/index", :locals => {:guis_info => guis_info}
```

Comment:
not sure if this is the best way to pass variables into haml. let me know if there's a better way

Reply:
Everything has been manage is DB until now. I don't know enough of rails to correctly describe and manage a change in DB schema. The processing of this file must be done in the git repo and it is not supposed to be run periodically on the server.

Reply:
I understand your point of view. If we decide to go that route it will take a bit longer, I'm also not very familiar with rails or the git-scm DB